ext/src/ruby_api/func_type.rs

```diff
@@ -26,14 +29,33 @@ impl FuncType {
 }
 
 impl FuncType {
+    /// @yard
+    ///
+    /// A descriptor for a function in a WebAssembly module.
+    /// WebAssembly functions can have 0 or more parameters and results. Each param
+    /// must be a valid WebAssembly type represented as a symbol. The valid symbols are:
+    /// +:i32+, +:i64+, +:f32+, +:f64+, +:v128+, +:funcref+, +:externref+.
+    ///
+    /// @def new(params, results)
+    /// @param params [Array<Symbol>] The function's parameter types
```

Comment:
A small nit; noting it here but applicable widely: can we add some consistency around the ending period? In this case the `param` definition doesn't have one, but there are other cases in which there's one (same for the general description `def` and `see` entries).

Reply:
Absolutely! I noticed the same as I proofread. I added period everywhere except when it's only ~2 words:

https://github.com/bytecodealliance/wasmtime-rb/blob/0d5644ed7bbf1c368bcfea034e253f380d2f7cdc/ext/src/ruby_api/linker.rs#L79-L80

Arguably these read as `The module name.` and have a period. 🤷 no strong opinion on my end.

Reply:
For the `@see`, I'd be tempted to **not** end with a period. It gets rendered as a bullet point and it's not really a sentence:

<img width="620" alt="Screenshot 2022-11-11 at 7 00 19 PM" src="https://user-images.githubusercontent.com/224311/201445918-4cabd7a1-f157-47f8-aa1a-6407cad7c7a6.png">

-----


Agreed?

Reply:
> I'd be tempted to not end with a period. It gets rendered as a bullet point and it's not really a sentence:

Oh yeah in this case it makes sense, this just denotes my lack of experience with YARD 😄  I'm good with that, generally it's just to enforce some baseline consistency.